sdk/clientcore/http-jdk-httpclient/src/test/java/io/clientcore/http/jdk/httpclient/JdkHttpClientSingletonTests.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.http.jdk.httpclient;
+
+import io.clientcore.core.http.client.HttpClient;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+@Execution(ExecutionMode.SAME_THREAD) // To avoid a race condition because of singleton http client.
```

Comment:
Why is this needed? Since we expect customer apps also to be multithreaded, these tests should be able to handle multithreading.

Reply:
I copied the tests over from Azure Core. I can remove it and see how the tests behave.